django-data/image/common/constants.py

```diff
@@ -173,3 +173,18 @@ class SAMPLE_STORAGE_PROCESSING(EnumMixin, Enum):
 DAYS = TIME_UNITS.get_value('days')
 
 KNOWN_STATUSES = ['Error', 'Issues', 'Known', 'Pass', 'Warning']
+
+# TODO add all possible messages
```

Comment:
In addition to offending field in current implementation I show helping fields (help user to decide which value to provide for offending field). So I agree that It would be great to store offending and helping fields in model. Also I could add new method (uid2biosamples for example) for Animal and Sample models that will convert human readable name to model name, and opposite. What do you think?

Reply:
Done